fortigate_connector.py

```diff
@@ -800,6 +818,9 @@ def _get_params(self, param):
         ip_addr = param[FORTIGATE_JSON_IP]
         policy = param[FORTIGATE_JSON_POLICY]
 
+        # Need to check if exists due to existing playbook actions without the param (original dstaddr default)
+        address_type = param.get(FORTIGATE_JSON_ADDRESS_TYPE, FORTIGATE_DSTADDR_TYPE)
```

Comment:
Wouldn't adding a `default` value of `dstaddr` solve this and make it clearer to the user what's going on?

Reply:
So the `default` is set to `dstaddr`, however this is a lower check and confirmation for safety. Some older existing playbooks won't send the parameter at all (due to existing playbook code) so this will set a default at the app level for extra safety.